homeassistant/components/homematicip_cloud/device.py

```diff
@@ -6,21 +6,13 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-ATTR_CONNECTED = 'connected'
```

Comment:
Are these leftovers from previous refactor?

Reply:
Yes, there are multiple attributes, that never have been used.